### PR TITLE
Allow Styles To Be Used in Config

### PR DESCRIPTION
**Pull request by auisgold**
## Description
Made change so that you can put time of day styles in config rather than through tileserver, this way you can just put "#(style)" in your template rather than praying you get the if else string right in your template.

## Motivation and Context
Makes it so you only have to change your config when you want to change styles, rather than changing every template to use new styles when they come out. 

## How Has This Been Tested?
Can be tested by adding the appriopriate styles names to the config in the geocoding section and changing templates to have 
"style": "#(style)"

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ X ] My code follows the code style of this project.
- [ X ] My change requires a change to the documentation.
- [ X ] I have updated the documentation accordingly.

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -449,6 +449,10 @@
       "providerURL": "",
       "forwardOnly": false, // When true, disable reverse geocoding lookup
       "cacheDetail": 3,     // number of decimal places of lon/lat to use while caching geocoding (default 3 - or use 4 for 100x more detail)
+      "dayStyle": "",
```
Comment (jfberry) [inline config/default.json:452]: Should provide a comment here to explain what these are for

File: src/controllers/gym.js [hunk 2]
```diff
@@ -178,7 +178,22 @@ class Gym extends Controller {
 
 					require('./common/nightTime').setNightTime(data, conqueredTime)
 
-					await this.getStaticMapUrl(logReference, data, 'gym', ['teamId', 'latitude', 'longitude', 'imgUrl'])
+					data.style = "klokantech-basic";
+
+					if (data.dawnTime && this.config.geocoding.dawnStyle != ''){
+						data.style = this.config.geocoding.dawnStyle;
```
[the remark this answers is not in the document]
Comment (jfberry) [reply src/controllers/gym.js:184]: I also think you could move this to common, and call a routine passing in `this.config` ...

**Comment by jfberry** [comment]
This looks ready to go - please DM me in discord to confirm status with regard to testing

**Comment by jfberry** [comment]
requires merge to develop or rebase

**Comment by auisgold** [comment]
Ready to go

**Comment by auisgold** [comment]
Added comment in default.json. Best to keep default styles to be more beginner-friendly IMO.

**Pull request merged**